acceptance/tests/apply_plan_ssh.rb

```diff
@@ -7,7 +7,6 @@
   extend Acceptance::BoltCommandHelper
 
   ssh_nodes = select_hosts(roles: ['ssh'])
-  skip_targets = select_hosts(platform: [/debian-8/])
```

Comment:
Since this var is no longer defined we should remove the `if skip_targets.any?` bit below.

Reply:
Damn, eagle-eye Beech!